src/node/maa.h

```diff
@@ -0,0 +1,417 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Can we have a ticket documenting the temporary existence of this file and the condition that needs to be met for it to be removed (ideally with links to the relevant OE issue(s))?

Reply:
https://github.com/microsoft/CCF/issues/1546